benchadapt/python/benchadapt/adapters/_adapter.py

```diff
@@ -1,15 +1,28 @@
 import abc
 import datetime
+import logging
 import subprocess
 import uuid
 from typing import Any, Dict, List
 
 import requests
 from benchclients.conbench import ConbenchClient
-from benchclients.logging import fatal_and_log, log
+from benchclients.logging import fatal_and_log
 
 from ..result import BenchmarkResult
 
+log = logging.getLogger("benchadapt.adapters")
+
+# `basicConfig()` does nothing if the root logger already has handlers
+# configured (that is, if this benchadapt library is imported in the context
+# of a program that already has a root logger with handlers set up, then the
+# following call is a noop.
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s.%(msecs)03d %(levelname)s: %(message)s",
+    datefmt="%y%m%d-%H:%M:%S",
+)
```

Comment:
This takes effect if the benchadapt library used from within any program that is not `benchclients`.